### DISCUSSION TITLE
Reflections

### DISCUSSION THREAD

**Discussion by kaasym** [Universite-Gustave-Eiffel/NoiseModelling#609] [Q&A]
Hello, are the values of LAeq calculated for reference points without reflection from the building of the facade? (reflections from the considered facade), or are the calculated values including reflection from the facade? 

I think there is something wrong with the groovy scripts. The Building_Grid script creates the BUILD_PK field. On the other hand, the Building_3Dgrid creates the PK_BUILDING field (but there is something wrong with the PK field in the Building_3Dgrid script - it is necessary to add the PK field again using the script). After renaming the BUILD_PK field to PK_BUILDING, the corrections for reflection from the facade are calculated correctly. 

Thank you. Jirka

**Comment by nicolas-f** [comment]
Hi,

Propagation code does not take account of receiver building id. So the calculated values include the reflection of the facade.

There was some changes recently with receivers Buildings 3d code, I don't know if there is a new issue or something has been solved.

**Comment by kaasym** [comment]
Thank you for your answer. Is there any way to eliminate facade reflections?
Thank you for your answer. Is it possible to eliminate facade reflections? Is it significant that the Building_Grid script creates the BUILD_PK array and the Building3D script creates the PK_BUILDING array?
